packages/foreman/rubygem-globalid/rubygem-globalid.spec

```diff
@@ -5,24 +5,19 @@
 %global gem_name globalid
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.4.2
-Release: 2%{?dist}
+Version: 1.0.0
+Release: 1%{?dist}
 Summary: Refer to any model with a URI: gid://app/class/id
 Group: Development/Languages
 License: MIT
 URL: http://www.rubyonrails.org
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
```

Comment:
It is missing the `ruby(release)` requirement that we used to have
```console
$ rpm -q --requires rubygem-globalid-1.0.0-1.el8.noarch.rpm
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsXz) <= 5.2-1
ruby >= 2.5.0
ruby(rubygems)
rubygem(activesupport) >= 5.0
```

Reply:
Ahh, I keep forgetting what is and why its needed.

Reply:
Ah, very good. Thanks for that.